### DISCUSSION TITLE
local.ERROR: Allowed memory size of 536870912 bytes exhausted (tried to allocate 32768 bytes

### DISCUSSION THREAD

**Discussion by webdevmahabub** [leokhoa/laragon#385] [General]
local.ERROR: Allowed memory size of 536870912 bytes exhausted (tried to allocate 32768 bytes) {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Allowed memory size of 536870912 bytes exhausted (tried to allocate 32768 bytes) at E:\\laragon\\www\\services\\storage\\framework\\views\\5a5e8ec0abbcdbe5f54a9d274ced5cd27ebb6ebc.php:1)

**Comment by ghost** [comment]
reace up your memory-limit in Menue -> php -> quicksettings

**Comment by Jayen16** [comment]
any update?

**Reply by AWqxKAWERbXo** [reply]
Usually, I ran into memory limits while creating an infinite loop.
